docs/rest_api.md

```diff
@@ -479,30 +484,42 @@ Authorization requirements:
 
 `POST /node/<node>/power_cycle`
 
+Request Body:
+
+    {
+        "force": <boolean> (Optional, defaults to False)
+    }
+
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
 Accepts one optional boolean argument that determines whether to soft (default)
 or hard reboot the system.
 
-Request body:
-    {
-    	"force": <boolean> (Optional, defaults to False)
-    }
+Authorization requirements:
+
+* If the owner is a project, access to that project is required.
+* Otherwise, administrative access is required.
 
 #### node_set_bootdev
```

Comment:
We should probably add something here about the possible values for bootdev.

Reply:
it's already there https://github.com/CCI-MOC/hil/blob/master/docs/rest_api.md#for-ipmi-devices

Reply:
Ah, fair enough.